xml_inscriptions/transcribed/CA.Malibu.JPGM.L.75.AD.38.xml

```diff
@@ -181,11 +182,22 @@
             <p/>
          </div>
          <div type="commentary">
-            <p/>
+            <p>
+               <lb n="1"/>form: perforated orbiculus, located at the top of the stamp extending through two lines of text and into the center of the stamp
+               <lb n="2"/>BDRCO 531
+               
+               <lb n="3"/>     "broken on all sides resuling in damage to outer edges of stamp and loss of line beginning"
```

Comment:
I'd put this information back where it was originally; detail on condition belongs better in that area of the metadata rather than the commentary, esp. since there is a designated place for it in the template already.

Reply:
lb 1 and 2 refer to the "ideal" stamp (i.e. the original die) and so belong (for now) in the commentary, until we have a more refined set of form types. lb 3 refers to this particular example and so belongs in the 'condition' metadata.